### PR TITLE
Update deploy script

### PR DESCRIPTION
**Pull request by zhusee2**
# Purpose
Update deploy flow and use Travis CI's Github Pages plugin for deploying storybook

# Changes
- Use Yarn 1.7.0 on Travis CI
- Skip git operation on bumpversion script
- Deploy storybook with Travis CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,15 +67,21 @@ Gypcrete does not publish develop builds to the `dist` branch anymore. It now pu
...
+   ```
+   This will run `lerna publish`, which updates all `package.json` files in `packages/`.
 
-This will run `lerna publish`, which updates all `package.json` files in `packages/`, commits the changes with pre-configured message and tags that commit.
+5. Commit above changes, then create a pull request for this release branch.
+6. Create a new release on Github once it's merged into `master`.
 
-Then create a pull request for this release branch.
-Once it's merged into `master`, it should trigger `yarn release` and publishes packages to npm.
+At the time relase PR is merged, it should trigger `yarn release` on Travis CI and publishes packages to npm.
```
Comment (garfieldduck) [inline README.md]: Maybe we could add a reminder after the release complete:

```
Remember to create a new backport branch from master, e.g., `backport/x.y.y`, then submit a PR from the branch and merge it back to the `dev` branch.
```

File: README.md [hunk 2]
```diff
@@ -67,15 +67,21 @@ Gypcrete does not publish develop builds to the `dist` branch anymore. It now pu
   * When pushed to `master` branch --> publish a relase build
 
 ### Releasing
-When releasing a new build for Gypcrete, create a release branch `release/x.y.z` and bump version first by:
-```sh
-yarn bumpversion
-```
+When releasing a new build for Gypcrete, follow the steps:
```
Comment (garfieldduck) [inline README.md]: This guide for the releasing steps are super helpful ❤️

File: README.md (hunk 1 above)
Comment (zhusee2) [reply README.md]: Sounds great, but how do you want to remind?

Comment (zhusee2) [reply README.md]: Nice catch

**Pull request merged**